ambry-network/src/main/java/com/github/ambry/network/NettyServerRequestResponseChannel.java

```diff
@@ -49,11 +52,10 @@ public void sendResponse(Send payloadToSend, NetworkRequest originalRequest, Ser
       throw new IllegalArgumentException("NetworkRequest should be NettyRequest");
     }
 
-    RestResponseChannel restResponseChannel = ((NettyServerRequest) originalRequest).getRestResponseChannel();
-    restResponseChannel.setHeader(RestUtils.Headers.CONTENT_LENGTH, payloadToSend.sizeInBytes());
```

Comment:
Where is the CONTENT_LENGTH header set now since it's being removed from here?

Reply:
It was used because we reused frontend NettyMessageProcessor which has a dependency on CONTENT_LENGTH. Since we totally dropped NettyMessageProcessor, CONTENT_LENGTH is no longer needed.